frontend/src/Explorer/Util/Time.purs

```diff
@@ -29,3 +38,20 @@ prettyDuration lang dur | convertDuration dur < Minutes 1.0 = "< 1 " <> translat
         if nu == 0
             then ""
             else show nu <> " " <> st
+
+nominalDiffTimeToDateTime :: NominalDiffTime -> Maybe DateTime
+nominalDiffTimeToDateTime (NominalDiffTime s) =
+    map toDateTime <<< instant $ fromDuration s
+
+
+type TimeFormat = String
+
+prettyDate' :: TimeFormat -> DateTime -> Maybe String
```

Comment:
You are right. I was not sure if we will need `TimeFormat -> Second -> Maybe String` anytime, so with `prettyDate` can provide it. To prevent it for using outside from `Time.purs` it is not exposed yet.